### PR TITLE
Update zh_hk.json to newest version.

### PR DESCRIPTION
**Pull request by SistineFibelKen**
## Description

update Hong Kong (Cantonese) language translation.

## (Optional) screenshots / videos

It may hard to translate the text into english with Google Translation.

## For Chinese user 对中国用户的一些话

这里的翻译后的语言均是粤语，因此有些仅看的懂的普通话用户是无法理解的，因此请不要瞎提出意见或者乱指正。
如果您懂得粤语或者来自中国香港的用户，欢迎来纠正译文错误，感谢。:)
(In order made Alexander01998 easy to understand : The translated language here is Cantonese, so some putonghua only users can't understand, please do not make any mistake comments or corrections.
If you know Cantonese or from Hong Kong, China, welcome to correct the translation error, thank you.:) )

### CODE REVIEW

File: src/main/resources/assets/wurst/lang/zh_hk.json
Comment (Jerrylum) [inline src/main/resources/assets/wurst/lang/zh_hk.json:87]: it should be: `生成隨機嘅物品然後掉喺地上。\n§o僅限創造模式。§r`

**Review by Jerrylum — CHANGES_REQUESTED** [review]
I am Jerry. Nice to meet everyone here. Sorry for the late review. I live in Hong Kong and am a native speaker in Cantonese and Traditional Chinese. This is my first-time contributing to this project. Please tell me if I did something wrong. Thank you.

First, thank @SistineFibelKen for their help. I would like to suggest some corrections in order to make the sentences more in line with what most people use today in Hong Kong and to make the sentence more fluent. Everyone is welcome to have a discussion together.

In addition, please do me a favour and change all `服務器` to `伺服器`? It would be great if we can fix everything small problems in this pr. Either way, thank @Alexander01998 for the mod.

**Comment by SistineFibelKen** [comment]
多謝 @Jerrylum 嘅出手幫助，終於遇到一位香港人或講粵語嘅人了。我唔系香港嗰邊嘅，但我系廣州人，識講粵語但唔識拼寫粵語正字，因此以後請多多指教啦。:)

Thanks @Jerrylum Help me. Finally met a Hong Kong native people , I am not live in hongkong but i live in Canton . i can speak cantonese but i poor in spelling the Traditional Chinese(especially cantonese).So please give me more advice in the future. :)

**Comment by Jerrylum** [comment]
唔使咁客氣，話說你哋有冇交流群之類咁嘅嘢呢？（唔限中文）我發現呢度好多寫中文字嘅人，真係意想不到。

另外我亦都改咗少少嘢，但係因為我冇寫入權限，所以冇辦法 push 去你個 repo 嗰度，我將更改好嘅檔案儲存咗喺 `zh_hk.txt` （冇辦法，GitHub 唔畀我上傳 .json），麻煩你幫幫手將呢個檔案覆蓋舊嗰個然後 push 上嚟。你再 double check 多次，如果冇問題嘅話講聲，咁我就畀你過 review。唔該曬你。


You're welcome. Btw, do you guys have any groups? (not limited to Chinese) I found that there are a lot of people who can write Chinese, that's unexpected ngl.

Apart from that, I changed something in the file. However, I was unable to push it to your repository since I didn't have the write access. I saved the finished version to `zh_hk.txt` (GitHub didn't want me to upload .json). Please help me to replace the old file and push the changes again. Then, double-check and let me know me if it is okay. I will approve the review after that. Thank you so much.


[zh_hk.txt](https://github.com/Wurst-Imperium/Wurst7/files/8568239/zh_hk.txt)

**Comment by SistineFibelKen** [comment]
> 唔使咁客氣，話說你哋有冇交流群之類咁嘅嘢呢？（唔限中文）我發現呢度好多寫中文字嘅人，真係意想不到。
> 
> 另外我亦都改咗少少嘢，但係因為我冇寫入權限，所以冇辦法 push 去你個 repo 嗰度，我將更改好嘅檔案儲存咗喺 `zh_hk.txt` （冇辦法，GitHub 唔畀我上傳 .json），麻煩你幫幫手將呢個檔案覆蓋舊嗰個然後 push 上嚟。你再 double check 多次，如果冇問題嘅話講聲，咁我就畀你過 review。唔該曬你。
> 
> You're welcome. Btw, do you guys have any groups? (not limited to Chinese, or Discord server idk) I found that there are a lot of people who can write Chinese, that's unexpected ngl.
> 
> Apart from that, I changed something in the file. However, I was unable to push it to your repository since I didn't have the write access. I saved the finished version to `zh_hk.txt` (GitHub didn't want me to upload .json). Please help me to replace the old file and push the changes again. Then, double-check and let me know me if it is okay. I will approve the review after that. Thank you so much.
> 
> [zh_hk.txt](https://github.com/Wurst-Imperium/Wurst7/files/8568239/zh_hk.txt)

我唔清楚有冇交流群呢個嘢。我從zh_cn開始翻譯，後尾發現越嚟越多人翻譯zh_cn lang 我就唔參與 zh_cn 嘅翻譯轉為 zh_hk lang 嘅翻譯了。好多人都系提交了push之後就唔改了。如果你方便嘅話，我可以加你Discord 好友或wechat其他都德。

基本上冇問題嘅啦，已經覆蓋源文件啦，感謝你嘅付出。

Idk have any group about wurst, I started to translate zh_cn and later found that more and more people were translating zh_cn, Instead of participating in the translation of zh_cn, translate zh_hk. A lot of people submit a pull request and don't change wrong things . If it's convenient for you, I can add your friends' Discord or wechat or whatever.

In general, the translation is basically no problem, the source file has been overwritten, thank you for your efforts

**Review by Jerrylum — APPROVED** [review]
LGTM

Also my discord id: Jerrylum#0001

**Comment by SistineFibelKen** [comment]
Update Hong Kong (Cantonese) language translation.

**Review by Alexander01998 — APPROVED** [review]

**Comment by buiawpkgew1** [comment]
@SistineFibelKen 请问zh-hk 和zh-tw 有什么区别

**Pull request merged**
